libs/x-content/impl/src/test/java/org/elasticsearch/xcontent/provider/json/ESUTF8StreamJsonParserTests.java

```diff
@@ -22,6 +22,10 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Locale;
 
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Maybe just keep using `Matchers.*` instead of these static imports? I think this makes the PR easier to review.